### ISSUE TITLE
Dataset constraints for the Open division

### ISSUE THREAD

**Issue by psyhtest** [mlcommons/inference_policies#245]
For the upcoming v2.1 round, the old Object Detection benchmarks, based on the SSD-ResNet34 and SSD-MobileNet-v1 models and the COCO 2017 validation dataset, have been deprecated. The new Object Detection benchmark, based on the RetinaNet model and [a subset](https://github.com/mlcommons/training/tree/master/single_stage_detector#the-mlperf-subset) of the [Open Images v6](https://storage.googleapis.com/openimages/web/index.html) dataset.

According to the [relaxed rules](https://github.com/mlcommons/inference_policies/blob/master/inference_rules.adoc#412-relaxed-constraints-for-the-open-division) for the Open division:

> 2. The accuracy dataset must be the same as an existing Closed benchmark.
> ...
> 8. The model can be of any origin (trained on any dataset, quantized in any way, and sparsified in anyway).

This raises a question whether Object Detection models trained on the COCO 2017 dataset can still be used for submissions into the v2.1 round. According to 2, the answer is "no". According to 8, the answer is "yes".

**Comment by psyhtest** [comment]
The discussion is not restricted to Object Detection. In fact, we will be in the same situation with Image Classification if we switch the dataset to Open Images. The crux of the problem is that ImageNet and COCO are the most popular research datasets, with hundreds if not thousands of pretrained models available. In the case of a new dataset, such as the MLPerf subset  of Open Images, there initially will be only a single reference model, i.e. trained specifically on this dataset for MLPerf. That is, we risk breaking making useful comparisons with a large body of work.

When models operate on the same dataset, we can more readily establish their trade-offs. For example, MobileNet-v1 with the top1 accuracy of 71.676% is usually faster than ResNet50-v1.5 with the top1 accuracy of 76.456% when both are evaluated on the ImageNet validation dataset. The performance/accuracy tradeoff is clear. Suppose now that MobileNet-v1 exhibits the top1 accuracy of 91.234% on a different dataset. Then, the tradeoff is not clear anymore.

**Comment by psyhtest** [comment]
To provide a more useful comparison in the spirit of MLPerf, we can consider some of the following options:
1. Request that any models that submitters use in the Open division are trained or finetuned on the same dataset as used by the Closed division model.
1. Request that the Closed division model is additionally trained or finetuned on the dataset that submitters use in the Open division, and is also submitted to the Open division on the same dataset . For example, a submitter that wishes to compare the new RetinaNet model with any COCO models must finetune it on COCO as well, and make the finetuned RetinaNet model part of their submission too.

The first option is likely to be prohibitive in terms of additional effort and resources. For example, Krai routinely submit about a hundred Image Classification models and a dozen of Object Detection models. The focus also gets shifted away from inference to training.

The second option only requires training or finetuning a single model, instead of many. In fact, in some cases, a pretrained model can already be widely available: for example, even if ResNet50 gets retrained on Open Images for the Closed division one day, we can still use the same tried-and-tested ResNet50 model trained on ImageNet that we have been using since v0.5. I believe NVIDIA might already have RetinaNet trained on COCO.

**Comment by rnaidu02** [comment]
@petermattson @johntran-nv  Any feedback on the comment from David on allowing new datasets for Open submission?

**Comment by petermattson** [comment]
They're allowed in Open in Training. Personally, I believe dataset
construction is an important area of innovation and would allow them but
try and improve how they're disclosed on the Open results sheet. However,
this is a WG decision. :-)
